src/app/movie-list/movie-list.component.html

```diff
@@ -0,0 +1,29 @@
+<div class="row">
+  <div *ngFor="let movie of movie_list | async">
+      <div class="col s4 m2">
```

Comment:
This markup is broken for different screen widths 
![image](https://user-images.githubusercontent.com/12671147/43575220-f4169ac2-964e-11e8-85a0-a35710dfe3ea.png) 
![image](https://user-images.githubusercontent.com/12671147/43575248-0a00aef4-964f-11e8-9d47-8ddb1c0ba4bf.png)